src/Cli/Microsoft.TemplateEngine.Cli/Commands/details/DetailsCommand.cs

```diff
@@ -17,7 +17,7 @@ internal class DetailsCommand : BaseCommand<DetailsCommandArgs>
             : base(hostBuilder, "details", SymbolStrings.Command_Details_Description)
         {
             Arguments.Add(NameArgument);
-            Options.Add(VersionOption);
+            // Options.Add(VersionOption);
```

Comment:
if you'd like to keep commented code, please specify the reason.  Otherwise, please remove the code.

Reply:
We are taking out the `--version` option while this issue is not solved (https://github.com/dotnet/templating/issues/6811) which will not be done soon. I think commenting it in this case is ok, since it would make it easier to activate it again once that issue is solved.